devicon.json

```diff
@@ -11289,5 +11289,24 @@
                 "alias": "plain"
             }
         ]
+    },
+    {
+        "name": "stackblitz",
+        "altnames": [],
+        "tags": [
+            "Full-Stack-Development",
+            "Online-IDE",
+            "IDE"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#1389fd",
+        "aliases": []
```

Comment:
Please move the `stackblitz` entry up, so that it fits in alphabetically. `devicon.json` should be ordered by the `name` attribute.

Reply:
Also there should be an alias to the `plain` version since the `original` version fits all the requirements to be a `plain` version.

So altnames should look like this:
```
aliases: [
    {
        "base": "original",
        "alias": "plain"
    }
]
```